### DISCUSSION TITLE
Assistant pane: add `/llm model` or `@model` command to change model mid conversation

### DISCUSSION THREAD

**Discussion by brendanator** [zed-industries/zed#28092] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I'd like to change model half way through a assistant conversation

I think it'd be especially useful for looking up obscure errors with perplexity before fixing with a better coding llm

I suggest either of:
- `/llm <model>`
- `@<model>`

### If applicable, add mockups / screenshots to help present your vision of the feature

`/llm <model>` example
<img width="635" alt="image" src="https://github.com/user-attachments/assets/f8a5f5c5-499e-4cf8-9b25-25cfbca368c6">

`@<model>` example
<img width="612" alt="image" src="https://github.com/user-attachments/assets/aeb406a6-d3e9-4646-ae1f-37c506424c40">

**Comment by danilo-leal** [comment]
Hey, thanks for the issue! You can change the model any time via the model selector—one chat can have gone through multiple models. It's not via a command as you're proposing, but I guess you're also proposing something else? Let me know! I'm not quite sure what you expect to get out of a `@perplexity`, for example.
